### DISCUSSION TITLE
Updating Cork?

### DISCUSSION THREAD

**Discussion by coreyallen** [buresdv/Cork#283] [General]
Saw there was a new version available (via macupdater) and couldn't figure out how to update the app.  Downloaded the version from the website thinking it may be the latest and now I'm back down to .1

**Comment by buresdv** [comment]
You can update the same way as any other Homebrew-installed app. If you didn't delete it, you should still have the latest version installed.

Open the latest version, and check if Cork is in the list of outdated packages:
![Snímek obrazovky 2024-03-21 v 20 37 15](https://github.com/buresdv/Cork/assets/22037369/1bb1cf28-4a98-46ca-a5b0-1c6c12293f22)
Then, you can click `Update`, or select only Cork to update only Cork and nothing else, and click `Update 1 Package`. After that, if it doesn' restart automatically, manually restart Cork and it will be the latest version

**Comment by timtrace** [comment]
I'm in the same situation - saw a 1.3.4 update available on MacUpdater, but Cork's About dialog said v0.1 was installed. It wouldn't find the 1.3.4 as an available upgrade. I removed the cask and the tap with the intention of reinstalling, but I can't find my original instructions - I'll send an email in a minute.

**Comment by buresdv** [comment]
I had another user saying that MacUpdates was claiming there was another version, but that's a mistake.

What happened is:
- There's a branch `temporary-licensing`, which will be the next version
- I already applied a version tag to it so I would remember that it's the correct branch
- MacUpdater took this tag thinking the version is already out, even though it isn't

I'm still waiting for translations before I can release the next version, so if MacUpdater is claiming there's an update, just keep in mind it's mistaken this time.

**Reply by timtrace** [reply]
I bet the MacUpdater team has their hands full, lol.  Thank you for the explanation. Much appreciated.

**Comment by coreyallen** [comment]
I"m getting conflicting info 
![CleanShot 2024-03-21 at 20 14 45](https://github.com/buresdv/Cork/assets/5347493/1db44fcc-aad0-4b8f-9292-c761dfa8a021)

**Reply by buresdv** [reply]
You're running the demo, which is version 0.1. It's detecting the full version that you have installed via Homebrew itself.

Delete the demo and reinstall/run the full version.

**Reply by coreyallen** [reply]
I only have one Cork in my applications folder. If I delete it, where will I find the app?
![CleanShot 2024-03-21 at 21 02 59](https://github.com/buresdv/Cork/assets/5347493/9c22c18c-8fa2-405a-a790-b95b8fccaf7e)

**Reply by buresdv** [reply]
You should uninstall it via Homebrew and then follow the install instructions again to install it

**Reply by coreyallen** [reply]
I can't seem to find the post purchase install instructions, is there an email I can use to get in touch with you directly?

**Reply by buresdv** [reply]
Of course, send me a message with the email you bought Cork with if you bought it via the website, or your GitHub email if you both it through here

**Reply by coreyallen** [reply]
Sent via mastodon thanks so much
